### ISSUE TITLE
Error compiling with Due

### ISSUE THREAD

**Issue by monoboy4** [forkineye/E131#6]
Hey I wrote some code using this Library.  It works great with a mega but i need better processing speed so i upgraded to a Due.  Now i have compiling issues.  any ideas?

**Comment by veecue** [comment]
Not at all, without seeing your code and compiler output. Please post it

**Comment by monoboy4** [comment]
Here's the error I'm getting with me compiling the Ethernet_test code.  The code compiles with no errors for an arduino mega 2560 but I get this error with the Due.  I get a similar error with the intel Galileo.   ![due error](https://cloud.githubusercontent.com/assets/18476319/14591562/a9394578-04e1-11e6-9f4d-21ea8051435c.png)
